src/yarn.ts

```diff
@@ -1375,78 +1353,72 @@ const completionSpec: Fig.Spec = {
       name: "workspace",
       description: "Manage workspace",
       generateSpec: async (_tokens, executeShellCommand) => {
-        const { postProcess } = scriptList;
-        const subcommands = [];
+        const version = await executeShellCommand("yarn --version");
+        const isYarnV1 = /^1\.\d+\.\d+/.test(version);
```

Comment:
nit: this is more performant
```suggestion
        const isYarnV1 = version.startsWith('1');
```